### ISSUE TITLE
"Signups are currently disabled"

### ISSUE THREAD

**Issue by gumatias** [CodeboxIDE/codebox#466]
I tried to sign up a week or so ago but no luck, even tried with different providers. I was wondering when people will start to be able to sign up again. 

Thanks for creating this amazing IDE! can't wait to try it out! :D

**Comment by tsaulic** [comment]
I'm wondering the same.. it doesn't look good that the last update was in Sep. 2014, though :(

**Comment by TylerJewell** [comment]
They announced about 6 months ago that they were no longer supporting the
project.

<http://codenvy.com> | Tyler Jewell | ceo | 978-884-5355

**Comment by aaronbartell** [comment]
I had correspondence with one of the committers about 1 month ago.  Where is it stated they are no longer supporting the project?

**Comment by TylerJewell** [comment]
It was one of either their LinkedIn profiles, a blog post, or on the site
itself.  I don't think it was the latter.

If you look at the two founders from Europe, their github commits have been
dedicated to gitbook over the past 6 months or so.

  <http://codenvy.com/> | Tyler Jewell | ceo | 978-884-5355

**Comment by gumatias** [comment]
what a bummer :( I guess I'll have to stick with c9.

**Comment by aaronbartell** [comment]
@SamyPesse or @AaronO can you give us some insight as to the future of codebox.io?

**Comment by rhutchison** [comment]
They are still actively charging my credit card, but it looks like it's time to find an alternative.  They didn't bother emailing their customers of any change.

**Comment by MitchK** [comment]
I think @SamyPesse and @AaronO are pretty much engaged with GitBook at the moment.
But I'm also curious on updates.

**Comment by aaronbartell** [comment]
I've been researching options for on-premise IDEs (a requirement for me) and found that [Eclipse Orion has a Node.js implementation](https://wiki.eclipse.org/Orion/Node/Getting_started).

**Comment by AaronO** [comment]
Hi Guys @MitchK  @rhutchison @aaronbartell @gumatias @TylerJewell,

I just wanted to give you guys a quick (and good) update about Codebox :)

As @TylerJewell mentioned, @SamyPesse and I are the team behind both [Codebox](https://www.codebox.io) ([CodeboxIDE/codebox](https://github.com/CodeboxIDE/codebox)) and [GitBook](https://www.gitbook.com]) ([GitbookIO/gitbook](https://github.com/GitbookIO/gitbook)), as a small team (it's hard to be smaller than 2 ;) ) time is a luxury, it's impossible to develop and maintain two great services with just the two of us.

A few months back we made the hard but necessary decision to pivot our core focus to **GitBook** whilst searching a home for Codebox. I'm happy to say that we found a great home just a few weeks ago ! (We're closing all the nitty gritty/legal details as we speak, so I can't disclose everything).

This company will be a great home for many reasons :
    * They are committed to keeping Codebox open source and keeping the community in the loop
    * They will be investing engineering effort in developing Codebox, both the IDE and service
    * *Slightly off topic*: From all my interactions, their management team seems to be composed of genuinely nice people :)

@SamyPesse and I will work closely with the company over the next few months, ensuring that Codebox is successfully on a new and bright path, giving Codebox the opportunity to grow and fulfill it's true vision.

This transition is great news, a new beginning for Codebox, an opportunity to build something truly great.

**Bonus:** As part of this transition, @SamyPesse will also be finishing off the `1.0.0` release ! :tada: 

**P.S:** I'm personally sorry for not updating you earlier, I hope you understand that it's been a crazy past few months.

**Comment by aaronbartell** [comment]
Thanks for the update @AaronO, I am happy for you guys.

>They are committed to keeping Codebox open source and keeping the community in the loop

Do you know if they plan on keeping the same license?

**Comment by AaronO** [comment]
@aaronbartell Thanks !

I can guarantee that it will be open source, but it's *possible* they may tweak the license. I think open source is a big part of Codebox's identity and one of it's core advantages. It's in Codebox's best interest to preserve and grow the open source community supporting it. They understand that just as well as I do.

**P.S:** @SamyPesse and I will continue on our GitBook adventure, with lots of really cool stuff coming :). Try it out if you haven't it's a great platform for documentation and collaborative books !

**Comment by OverflowCreative** [comment]
Anyone have any updates on codebox? I really want to try it out.

**Comment by sdillman** [comment]
"Bump."  I remain very interested in where Codebox ends up, and when I might be able to sign up for it.

**Comment by YNedderhoff** [comment]
Me as well, I'm regularly checking the codebox.io page for month now hoping for updates ...

**Comment by scrumit** [comment]
Is there an alternative? The code is open source. 

Perhaps it can be modified to connect to any cloud based platform as a service, like [this blog posting describes for another editor](http://simpleprogrammer.com/2014/10/13/getting-started-meteor-tutorial-cloud/).

Could the community that is using it, and wants to use it create a community facility? Does it need to go to commercial venture who will licence it?

**Comment by ghost** [comment]
I'd love to launch this as a service if there is no future for Codebox doing it themselves. Would anybody else be interested in helping out with this? We could possibly use a platform like https://assembly.com to make it a community project.

**Comment by Jaballadares** [comment]
Hey @emberbit I'd be interested in participating, while my coding skills are lacking I could help on the marketing front.

**Comment by ghost** [comment]
@Jaballadares - Nice, let's see if anyone else is interested. Post a new issue?

**Comment by thegecko** [comment]
It would be good to understand the creators plan for this product from a technical perspective, too.
I see PRs aren't being merged at the moment, so are there plans to fix/accept fixes for 1.0.0 issues (such as collaboration) in this repo any more?

**Comment by scrumit** [comment]
There are 320 forks of the CodeBoxIDE codebase. And there are a few Pull Requests outstanding. A lot of people would like to see CodeBoxIDE available.

However, it might better to start again. Anyone up for that? If you have made PRs then you know the features that you feel committed to.

I'm an architect and I always feel uncomfortable if I see no architectural description of a system. I'm been looking at [couchbase](https://www.joyent.com/blog/couchbase-in-docker-containers) recently and I wonder if a webIDE shouldn't have the following elements:
- SSH command line
- file navigator
- editor with pluggable tooltips/completion/
- devops facility to deploy systems at scale
- audio/visual collaboration

I suspect if everything is pluggable, then a JSON document store would be the most consistent way to store configurations. So I see basing an IDE on top of a scalable open source JSON document store changes quite a bit of how the system develops and deploys. If you have node.js at the server side then the system that makes the IDE work, with all its functionality can also be available at run time. Possibly the first truly Integrated develop (and run) Environment.

Thoughts? Is there another forum to take this?

**Comment by aaronbartell** [comment]
Another option is to take a look at [Eclipse Orion](https://wiki.eclipse.org/Orion/Node/Getting_started) - the version that uses Nodejs as its sever-side component.  It doesn't have nearly as many features, but does have a big(ger) body of support.

**Comment by MitchK** [comment]
Maybe a port of https://atom.io/ for the browser with a web service layer in between?

**Comment by despiegk** [comment]
We would be interested to push this project forward again.
We are even willing to fund the time it takes to do this.
I am looking for a lead on this, and can provide 1-2 developers 
- looking for interest from community.
  I am CEO of www.incubaid.com
  It would be great if we can get 
- this tool supported again
- add video & chat
- support docker & autobuild (we have some ideas here) on top of multiple clouds
- seamless ssh access to the dockers
- keep all opensource 
- all integrated with oauth2

**Comment by jmls** [comment]
does anyone have any success stories here ? Do you know of any open-source products that work in docker ? Seems such a shame that this project folded

**Comment by TylerJewell** [comment]
Might you consider Eclipse Che?

It's a new product that has been in a rewrite for 2 years - about 4,000
commits on the project with around 50 committers.

It is alpha now.

It is going to make it's first beta announce in a few weeks.  We are
looking for pre-release reviewers who want to offer some honest commentary.

Some of the unique qualities of Che:
1. Workspace portability.  (workspaces are combinations of projects +
machines, which are powered by docker containers - these can be snapshot &
moved to different locations)
2. Multi-project workspaces.
3. Multi-module projects, with drill-in / drill out capability.
4. Built in SSH workspaces
5. Terminal
6. Stacks - shipping with pre-defined, all-in-on, and custom stacks that
contain frameworks of software that will power the workspaces
7. Commands - context-sensitive commands that are injected into machines
8. Run the che server as a native server, or itself within a docker
container

If anyone was interested in getting a pre-release demo + explanation along
with instructions on how to get started with it, happy to provide it.

I think we'll be public with the beta & simple installers in about 3 weeks.

Tyler Jewell | CEO | tyler@​codenvy.​com | 9​78​.8​84​.53​55

**Comment by jmls** [comment]
yes, more than happy to work on the preview - tell me more ;) Emailed you my details. thanks.

**Comment by TylerJewell** [comment]
Awesome - great.  I didn't see the email show up.

But if anyone wants to write to learn more, can talk at tjewell@codenvy.com.
